dev/versions.md

```diff
@@ -112,39 +110,60 @@ It's probably safest to use the minor version of Python that you get when you ru
 1. Search the codebase for the previous ONNX runtime version
 
 ## Nvidia device plugin
 
 1. Update the version in `images/nvidia/Dockerfile` ([releases](https://github.com/NVIDIA/k8s-device-plugin/releases), [Dockerhub](https://hub.docker.com/r/nvidia/k8s-device-plugin))
-1. In the [GitHub Repo](https://github.com/NVIDIA/k8s-device-plugin), find the latest release and go to this file (replacing the version number): <https://github.com/NVIDIA/k8s-device-plugin/blob/1.0.0-beta/nvidia-device-plugin.yml>
+1. In the [GitHub Repo](https://github.com/NVIDIA/k8s-device-plugin), find the latest release and go to this file (replacing the version number): <https://github.com/NVIDIA/k8s-device-plugin/blob/v0.6.0/nvidia-device-plugin.yml>
 1. Copy the contents to `manager/manifests/nvidia.yaml`
    1. Update the link at the top of the file to the URL you copied from
    1. Check that your diff is reasonable (and put back any of our modifications, e.g. the image path, rolling update strategy, resource requests, tolerations, node selector, priority class, etc)
 1. Confirm GPUs work for PyTorch, TensorFlow, and ONNX models
 
 ## Inferentia device plugin
 
-1. Check if [k8s-neuron-device-plugin](https://github.com/aws/aws-neuron-sdk/blob/master/docs/neuron-container-tools/k8s-neuron-device-plugin.yml) has been updated since the last time (we're running the version listed here: https://github.com/aws/aws-neuron-sdk/issues/102). If so, then update `images/inferentia/Dockerfile` and update `manager/manifests/inferentia.yaml` with the latest and replace the container's image with `$CORTEX_IMAGE_INFERENTIA`. Currently, all device versions are residing at [robertlucian/cortexlabs-inferentia](https://hub.docker.com/repository/docker/robertlucian/cortexlabs-inferentia), because the ECR repo that was hosting the image seems to have been taken down. See https://github.com/cortexlabs/cortex/issues/1133.
+1. Check if the image in [k8s-neuron-device-plugin.yml](https://github.com/aws/aws-neuron-sdk/blob/master/docs/neuron-container-tools/k8s-neuron-device-plugin.yml) has been updated (also check the readme in the parent directory to see if anything has changed). To check what the latest tag currently points to, run `aws ecr list-images --region us-west-2 --registry-id 790709498068 --repository-name neuron-device-plugin`, and then see which version has the same imageDigest as `latest`.
+1. Copy the contents of [k8s-neuron-device-plugin.yml](https://github.com/aws/aws-neuron-sdk/blob/master/docs/neuron-container-tools/k8s-neuron-device-plugin.yml) and [k8s-neuron-device-plugin-rbac.yml](https://github.com/aws/aws-neuron-sdk/blob/master/docs/neuron-container-tools/k8s-neuron-device-plugin-rbac.yml) to `manager/manifests/inferentia.yaml`
+   1. Update the links at the top of the file to the URL you copied from
+   1. Check that your diff is reasonable (and put back any of our modifications)
+
+## Neuron RTD
+
+1. `docker run --rm -it amazonlinux:2`
```

Comment:
I decided to merge it into one section because the all should be run together, as a single "transaction", before running or testing anything. Do you think that makes sense?

Reply:
The thought wasn't about executing them separately, but to make it harder to skip a step by mistake. It's okay this way too - I don't have too much of a strong opinion.